packages/repo-doctor/src/rules/index.js

```diff
@@ -19,6 +20,10 @@ const setupRules = (config, args) => {
     }
     let ruleFn = ruleFns[ruleName]
 
+    if (ruleFn.configSchema) {
+      validate(ruleFn.configSchema, {}, ruleConfig)
```

Comment:
Not knowing schema-utils, I had to look up the docs to see that

- it throws an error if the config doesn't match
- the middle arg is an empty option

I think we could use a named var, and idk if we should do something with the exception, like wrapping it in a custom error message?

Reply:
- I wrapped the error message into a ConfigError and only print the message if the error from validation is a ConfigError
- The second argument to validate was unnecessary, so I removed it